auth/util/src/test/java/org/wildfly/security/auth/util/SimpleRegexRealmMapperTest.java

```diff
@@ -15,6 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
In general, it's good practice not to include additional unrelated changes in the same PR so it would be better to remove this unrelated whitespace change.

Reply:
Sure, thanks!